### DISCUSSION TITLE
Publish On Homebrew-Core

### DISCUSSION THREAD

**Discussion by ivaquero** [carapace-sh/carapace-bin#2563] [General]
I just submitted a PR on homebrew-core, https://github.com/Homebrew/homebrew-core/pull/195391

The PR has passed the CI, and after it is merged (maybe one day later), carapace can be installed through the official tap.

**Comment by ivaquero** [comment]
To meet the requirements of Homebrew official, I deleted the caveats block which prints the hints of completer setup. 

If you think the caveats block is nessessary, you may update the formula.

**Reply by rsteube** [reply]
Shouldn't be necessary. There's the help output and online documentation for that.

**Comment by rsteube** [comment]
Updated instructions: https://github.com/carapace-sh/carapace-bin/pull/2575
